src/index.ts

```diff
@@ -12,6 +12,10 @@ export { applyDocumentUpdateMessage, encodeDocumentUpdateMessage } from './messa
 export { encodeCompleteAwarenessStateRequestMessage } from './messages/complete-awareness-state-request-message'
 export { encodeCompleteDocumentStateRequestMessage } from './messages/complete-document-state-request-message'
 export { encodeCompleteDocumentStateAnswerMessage } from './messages/complete-document-state-answer-message'
+export { encodeDocumentDeletedMessage } from './messages/document-deleted-message'
+export { encodeMetadataUpdatedMessage } from './messages/metadata-updated-message'
+export { encodeServerVersionUpdatedMessage } from './messages/server-version-updated-message'
```

Comment:
These should not be in the "refactor" commit but rather in the "add new message types" commit. Otherwise building the refactor-commit alone won't work.